JsonLd/Serializer/ItemNormalizer.php

```diff
@@ -152,7 +152,7 @@ public function normalize($object, $format = null, array $context = [])
                 $subResource = $this->resourceResolver->getResourceFromType($collectionType)
             ) {
                 $values = [];
-                foreach ($attributeValue as $index => $obj) {
+                foreach ($attributeValue->getValues() as $index => $obj) {
```

Comment:
We could use Doctrine's `Collection` interface as it's in `doctrine/common` and it not related/dependent at all on any database-related stuff.